storage/aptosdb/src/state_store/mod.rs

```diff
@@ -583,7 +584,7 @@ impl StateStore {
         first_version: Version,
         expected_usage: StateStorageUsage,
         ledger_batch: &SchemaBatch,
-        sharded_state_kv_batches: &[SchemaBatch; 256],
+        sharded_state_kv_batches: &[SchemaBatch; NUM_STATE_SHARDS],
```

Comment:
`type SharedStateKvSchemaBatch = ... ` would be nice.

Reply:
Done